src/docstrings.jl

```diff
@@ -0,0 +1,71 @@
+
+const XGBOOST_DOCS_LINK = "https://xgboost.readthedocs.io/en/stable/index.html"
+const XGBOOST_PARAMS_DOCS_LINK = "https://xgboost.readthedocs.io/en/stable/parameter.html"
+
+
+"""
+$(MMI.doc_header(XGBoostRegressor))
+
+Univariate continuous regression using [xgboost]($XGBOOST_DOCS_LINK).
+
+# Training data
+In `MLJ` or `MLJBase`, bind an instance `model` to data with
+```julia
+m = machine(model, X, y)
+```
+where
+- `X`: any table of input features, either an `AbstractMatrix` or Tables.jl-compatible table.
```

Comment:
The present implementation and trait declarations indicate that only tables are allowed. (Although I'd be happy for this to be changed, if you wanted to.)

```suggestion
- `X`: any table of input features whose columns have `Continuous` element scitype; check 
  column scitypes with `schema(X)`.
```

The same suggestion applies to the other models but are not marked.